src/api/custom-schema.ts

```diff
@@ -128,8 +129,15 @@ export const extendGraphqlSchema = (schema: GraphQLSchema) =>
               data: exporterCompanyAddress,
             });
 
+            // maps SIC codes from variables into array
+            const sicCodesToAdd = mapSicCodes(company, sicCodes);
+            // TODO: delete existing SIC codes when adding sic codes to db
+            await context.db.ExporterCompanySicCode.createMany({
+              data: sicCodesToAdd,
```

Comment:
Could we call this `mappedSicCodes`?

Reply:
done

Reply:
Sorry I meant the const,  like this

```js

const mappedSicCodes = mapSicCodes(company, sicCodes);
```

Just we have this pattern in other places